### ISSUE TITLE
Cannot traverse from and to the same ID, ``.

### ISSUE THREAD

**Issue by timarney** [andrepolischuk/react-rotation#7]
I'm getting this error on my test demo

`invariant.js:39Uncaught Invariant Violation: traverseParentPath(...): Cannot traverse from and to the same ID, ``.`

using version React 15.0.1

Seems to be this line:
> var el = (0, _reactDom.findDOMNode)(this);  in index.es5.js how does this line work?  I haven't seen that syntax before.

**If I attach the listener events to the window - everything works i.e. var el = window;**

My code
```
import React from 'react';
import {render} from 'react-dom'
import Rotation from 'react-rotation'
const Hello = (props) =>
<div style={ {width:200} }>
    <Rotation>
        <img src='img/0000.jpg' />
        <img src='img/0001.jpg' />
        <img src='img/0002.jpg' />
        <img src='img/0003.jpg' />
    </Rotation>
</div>

render(
  <Hello name="Sebastian" />,
  document.querySelector('#app')
)
```

I setup a repo here https://github.com/timarney/spin 

=====================================

**Comment by andrepolischuk** [comment]
It was fixed by update react to 15.0 #8

**Comment by andrepolischuk** [comment]
Published as 0.4.0
